content/transactions/accountset.md

```diff
@@ -64,6 +64,7 @@ The available AccountSet flags are:
 | asfNoFreeze      | 6             | lsfNoFreeze               | Permanently give up the ability to [freeze individual trust lines or disable Global Freeze](concept-freeze.html). This flag can never be disabled after being enabled. |
 | asfGlobalFreeze  | 7             | lsfGlobalFreeze           | [Freeze](concept-freeze.html) all assets issued by this account. |
 | asfDefaultRipple | 8             | lsfDefaultRipple          | Enable [rippling](concept-noripple.html) on this account's trust lines by default. [New in: rippled 0.27.3][] |
+| asfDepositAuth   | 9 | lsfDepositAuth | Enable [Deposit Authorization](concept-depositauth.html) on this account. |
```

Comment:
If we're ordering by Decimal Value, that should probably be the first column. (Sorry, sorting is on my mind.)

Reply:
That's fair, but to be consistent with how I'm ordering the AccountRoot flags in the ledger reference, I'm going to sort them alphabetically by name rather than numerically.